tests/globalhelper/runhelper.go

```diff
@@ -47,5 +48,44 @@ func LaunchTests(testSuites []string, skipRegEx string) error {
 	cmd.Args = append(cmd.Args, testArgs...)
 	cmd.Dir = Configuration.General.TnfRepoPath
 
+	if os.Getenv("DEBUG_TNF") == "true" {
+		os.Setenv("LOG_LEVEL", "trace")
+
+		buildNumber, present := os.LookupEnv("BUILD_NUMBER")
```

Comment:
What is BUILD_NUMBER and why do we need it?

Reply:
This env var will be passed from Jenkins, and its the build number.
We need to be able to distinguish between ci logs, in order to see what went wrong in the specific build.

Reply:
Deleted it.